README.md

```diff
@@ -86,6 +86,57 @@ export const App = () => {
 
 [Seam Console]: https://console.seam.co/
 
+### React Hooks
+
+All components interact with the Seam API via a set of React Hooks.
+You can use these hooks directly in your application to build your own components.
+
+Import hooks directly from `@seamapi/react` or `@seamapi/react/hooks`.
+For most applications, these imports are equivalent,
+however if you are only using the hooks, we recommend importing from `@seamapi/react/hooks`
+as it may enable performance or bundling improvements depending on your build system.
+
+Hooks must be used inside the `SeamProvider`.
+They are well-typed and follow a uniform API.
+
+```ts
```

Comment:
Should this be `tsx` or is there no support for that?

Reply:
Oh nice! Let me try 👍🏻

Reply:
I was wondering why the highlighting was not perfect ❤️  [`1f2a944` (#260)](https://github.com/seamapi/react/pull/260/commits/1f2a9445863f955d132dc40628bcd49914e58418)

Reply:
Awesome, good to know!